### ISSUE TITLE
Test mode stuck in loading

### ISSUE THREAD

**Issue by jerfai** [FlutterFlow/flutterflow-issues#1823]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

Test mode is stuck loading and i'm unable to test.
![image](https://github.com/FlutterFlow/flutterflow-issues/assets/128825755/ae3e05c6-dbab-4a74-8525-494d746efe88)

### Expected Behavior

i'm expecting to see my app after the test mode has finished loaded

### Steps to Reproduce

1. open project
2. press test mode
3. wait until it launches in a new window
4. wait

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4siMmB7ZNPn7xK1br+YMJ7/SUUJjt9a+csscp/bxIgGIDzPOw+ZPWkVEpBQuHiYwhie0SmmmI10++LiOGXG/oVOReuQKp4+btUUAvveFiha5OBDM6KT0YlEcZjfG6i36SvvBFTJvJvW1Y532GpfdnIFyvvJ5C2IlFISq/LZO4=

### Context

i'm unable to test my app

### Visual documentation

image already provided, please check the console (screenshot from chrome). console errors :

Failed to load resource: the server responded with a status of 504 ()
require.js:143 Uncaught Error: Script error for "packages/power_shark_app/backend/push_notifications/push_notifications_util.dart", needed by: web_entrypoint.dart
http://requirejs.org/docs/errors.html#scripterror
    at makeError (require.js:168:17)
    at HTMLScriptElement.onScriptError (require.js:1738:36)
8Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
XpgvP5U9VPXLYxFPlAr0/:1 Access to internal resource at 'https://powersharkwebstatic.z33.web.core.windows.net/manifest.json%3E' from origin 'https://ff-debug-service-frontend-ygxkweukma-uc.a.run.app' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

### Environment

```markdown
- FlutterFlow version: 4
- Platform: windows
- Browser name and version: chrome latest
- Operating system and version affected: windows 11
```

**Comment by treirouxel88** [comment]
I also have the same problem here, i have a similar error about a json file

dart_sdk.js:5415 Uncaught (in promise) Error: FormatException: SyntaxError: Unexpected end of JSON input

at Object.createErrorWithStack (dart_sdk.js:5415:12)
at Error._throw (dart_sdk.js:8564:18)
at Error.throwWithStackTrace (dart_sdk.js:8561:18)
at async._AsyncCallbackEntry.new.callback (dart_sdk.js:40357:18)
at Object._microtaskLoop (dart_sdk.js:40214:13)
at _startMicrotaskLoop (dart_sdk.js:40220:13)
at dart_sdk.js:35968:9

**Comment by devjones** [comment]
Can confirm this is a major issue.

**Comment by dsroy** [comment]
I had the same problem.  I believe it started when I manually cleared the browser's local storage of the auth variables to reset my app.  I got the app working again by manually clearing each item in the local storage of the app.  I'm using Chrome and cleared the storage by right clicking on the storage item and selecting "clear".  I hope this helps.  Happy to provide more info as needed.

**Comment by agreaves** [comment]
Hey folks, thanks for the bug report and info! There are a lot of possible issues within the app that could cause it to load forever, and I think the issues described here may be different from each other. All it means is that something is failing when the app is initialized / first loaded, and that could happen in a number of ways.

In the original case of this issue, it appears to be an error in the On Page Load action due to CORS configuration on the API. You may need to whitelist `https://ff-debug-service-frontend-ygxkweukma-uc.a.run.app` in order to make it work. Does this also happen when you run via Run Mode?

**Comment by hariprasadms** [comment]
Hi, Due to inactivity and no further follow, the current issue is closed. If the problem persists, kindly initiate a new issue and provide all necessary information. Thank you.
